openpype/hosts/fusion/plugins/publish/extract_review_data.py

```diff
@@ -0,0 +1,55 @@
+import os
+import pyblish.api
+from openpype.pipeline.publish import (
+    ColormanagedPyblishPluginMixin,
+)
+
+
+class FusionExtractReviewData(
+    pyblish.api.InstancePlugin, ColormanagedPyblishPluginMixin
+):
+    """
+    Extract the result of savers by starting a comp render
+    This will run the local render of Fusion.
+    """
+
+    order = pyblish.api.ExtractorOrder - 0.1
+    label = "Extract Review Data"
+    hosts = ["fusion"]
+    families = ["review"]
```

Comment:
Shouldn't this be on the "render" family? This doesn't feel like review data but this also now contains the actual file representation data of the render data itself.

The way this is separated into its own plugin feels quite weird to me. Need to read more thoroughly to really understand why it needs to be separate.

Reply:
But if you publish using existing frames you need to get that file-array some how. Should that be in a third plugin then maybe?

Reply:
Since you'd also need to validate they actually exist - I'd actually create a Collector which would collect the expected frames/files, the validator would validate those pre-collected frames and the extractor I guess even the one that's set to 'render local' could just add the representation with those frames (and only render if it's set to "render")?

Reply:
That's a better way actually! Implemented :)